bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -62,17 +71,34 @@ def kfcv_awkde(sample, bwchoice, alphachoice, k=2):
 
 
 def optimizedparam(sampleval, nfold=2):
-    # Test wide range of bandwidths in addition to standard values
-    bwgrid = ['scott', 'silverman'] + numpy.logspace(-2,0,10).tolist()
-    # Fix adaptive sensitivity parameter to 1
-    alphagrid = [1]
+    # bw-grid and alpha-grid are values given as arguments to the function
```

Comment:
this comment is not correct. The arguments of the function are what variables are passed to it inside the brackets, currently these are 'sampleval' and 'nfold'. To make the function self-contained 'bwgrid' and 'alphagrid' should also be arguments of the function. This is not connected with argparse (command line arguments of the whole script)